wordpress-mobile/WordPress-Android
# Feature/4263 replace optimizely with mixpanel

Fixes #4263 

Well that was easy...

One thing we do lose is Tracks integration, but since we are using Mixpanel for analytics anyways, it's not a big deal.

To test:

Check out Mixpanel and see that there is a "Magic Link" experiment and that Variation 1 has `showMagicLink` boolean set to true.

Needs review: @maxme

cc: @aerych
